justfile

```diff
@@ -43,6 +46,10 @@ build-binary:
   GOOS=linux go mod vendor
   GOOS=linux go build -v
 
+build-binary-eks:
+  GOOS=linux go mod vendor
+  GOOS=linux GOARCH=amd64 go build -v
```

Comment:
what makes it an EKS binary? I don't think we need to add another specific command like this one, especially when you can add a parameter to the original command for the `OS` and `ARCH`

Reply:
How can you add the params to the JUST `go build` command only?

I just thought it would be easier if we would have an easy command that knows how to replace EKS because the current command doesn't work.
I wouldn't expect anyone that hasn't worked on EKS to know that this is necessary for it to run.

If you still find it redundant after what I wrote I'll remove it.

Reply:
you can add params to any just command, for example:

```justfile
my-command param_1 param_2:
  command p1={{param_1}} p2={{param_2}}
```

not only that, parameters prefixed with a `$` will be exported as environment variables. See more [here](https://github.com/casey/just#getting-and-setting-environment-variables)

Reply:
So I tried two alternatives:

1. Concatenating parameters -  
```justfile
build-binary GOARCH='':
  GOOS=linux go mod vendor
  GOOS=linux {{GOARCH}} go build -v
```

But I couldn't find a way to run `build-replace-cloudbeat` with a parameter.
For example, this code doesn't work:
```justfile
build-replace-cloudbeat GOARCH='' : build-binary {{GOARCH}} 
  ./scripts/remote_replace_cloudbeat.sh
```

2. I can add an environment variable to the entire JUST file, and then I can use it in the script.
But I am wondering how people will know to change it.

I thought about maybe doing something like
# For an EKS cluster please set the GOARCH as 'GOARCH=amd64'
GOARCH := ''

WDYT? Do you have a better suggestion?

Reply:
> not only that, parameters prefixed with a $ will be exported as environment variables. See more [here](https://github.com/casey/just#getting-and-setting-environment-variables)

Reply:
reverted